README.md

```diff
@@ -266,6 +266,11 @@ else
 end
 ```
 
+The redirect after the new/edit is set to default to the index path of the created/updated resource, e.g.
```

Comment:
could this be a subsection in ### configure the engine? or have a heading of some kind to make it a bit easier to find?

Reply:
Heading added